core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -235,6 +237,14 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
     // variable in place of the 'this' keyword.
     let that = this;
 
+    let currentPath = this.urlService.getPathname();
+    let backpathUrl = this.PAGES_WITHOUT_BACK_STATE.find(
+      PAGES_WITHOUT_BACK_STATE => currentPath.includes(PAGES_WITHOUT_BACK_STATE)
```

Comment:
Just a general note for the future, please avoid having function parameters share the same name as variables in the outer scope.